skimage/measure/_regionprops.py

```diff
@@ -872,6 +1028,20 @@ def regionprops(label_image, intensity_image=None, cache=True,
     >>> props[0]['centroid']
     (22.72987986048314, 81.91228523446583)
 
+    Add custom measurements by passing functions as ``extra_properties``
+    >>> from skimage import data, util
+    >>> from skimage.measure import label, regionprops
+    >>> import numpy as np
+    >>> img = util.img_as_ubyte(data.coins()) > 110
+    >>> label_img = label(img, connectivity=img.ndim)
+    >>> def pixelcount(regionmask):
+    >>>     return np.sum(regionmask)
```

Comment:
This is causing the CI failure
```suggestion
            return np.sum(regionmask)
```

Reply:
Thanks a lot for finding this. I didn't look at the failing Travis build log as it has been failing with matplotlib for a while and assumed that was still the case. Will commit your suggestion.
I'm curious: are the other CI builds not building documentation or why are they all give the green tick?

Reply:
>  I'm curious: are the other CI builds not building documentation or why are they all give the green tick?

They are, and they crashed now :rofl: